operators/src/processing/meteosat/radiance.rs

```diff
@@ -0,0 +1,502 @@
+use crate::engine::{
+    ExecutionContext, InitializedRasterOperator, Operator, QueryContext, QueryProcessor,
+    RasterOperator, RasterQueryProcessor, RasterQueryRectangle, RasterResultDescriptor,
+    SingleRasterSource, TypedRasterQueryProcessor,
+};
+use crate::util::Result;
+use futures::stream::BoxStream;
+use futures::StreamExt;
+use geoengine_datatypes::primitives::SpatialPartition2D;
+use geoengine_datatypes::raster::{
+    EmptyGrid, Grid2D, GridShapeAccess, Pixel, RasterDataType, RasterPropertiesKey, RasterTile2D,
+};
+use num_traits::AsPrimitive;
+use std::convert::TryFrom;
+
+use async_trait::async_trait;
+use serde::{Deserialize, Serialize};
+
+//TODO Check
+type PixelOut = f32;
+use RasterDataType::F32 as RasterOut;
+use TypedRasterQueryProcessor::F32 as QueryProcessorOut;
+
+//TODO: For NAN the tests fail due to == comparisons
+const OUT_NO_DATA_VALUE: PixelOut = PixelOut::NEG_INFINITY;
+
+#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
+#[serde(rename_all = "camelCase")]
+pub struct RadianceParams {}
+
+#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
+pub struct RadianceState {
+    //TODO: Could be removed, if no data is a constant
+    out_no_data_value: PixelOut,
```

Comment:
I guess, since we have always float as output, we can just always use NAN.

Reply:
Applied in: 3be3fd383500c6069bb49ffde1523b83a9c4b184